client/packages/system/src/Patient/DetailView/DetailView.tsx

```diff
@@ -111,7 +121,12 @@ export const PatientDetailView: FC = () => {
             </TabList>
           </Box>
         </AppBarTabsPortal>
-        <PatientTab value={Tabs.Details}>{JsonForm}</PatientTab>
+        <PatientTab value={Tabs.Details} padding={3}>
```

Comment:
there is padding by default - I've removed it because it looks bad for the list view tabs. should only be needed on this first tab